node/core/proposer/Cargo.toml

```diff
@@ -23,4 +23,5 @@ sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sp-inherents = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch = "master" }
+tokio = "0.2.21"
```

Comment:
Within Substrate we try to use https://github.com/tomaka/wasm-timer/ for all timing related Futures to be able to run within any wasm environment. Would the same be needed here as well?

Reply:
Also this is missing the `time` feature flag, no?

Reply:
This is node code; I don't expect it ever to need to be compiled to WASM. That said, I'm happy to switch to the project standard.

Reply:
As far as I know we would like to run node code in the browser and thus need it to be able to run in WASM. Adding @tomaka here for more insights.

Reply:
We indeed compile the node code to Wasm.
And the standard is indeed the `wasm-timer` library.

Aside from compiling to Wasm, directly depending on `tokio` is to me unidiomatic considering that the code will panic if you don't use tokio for polling, and using tokio for polling is normally an implementation detail.